src/main/java/com/google/alpollo/SongDataBase.java

```diff
@@ -16,4 +17,14 @@ public static void saveSongRequest(Song song) {
   public static List<Song> topSongs() {
     return OfyService.ofy().load().type(Song.class).limit(TOP_SIZE).list();
   }
+
+  /** Save whole song info to database. */
+  public static void saveSongInfo(SongInfo songInfo) {
```

Comment:
I think you should also return the id of the song entity after saving it, since we'll need it for retrieving the song info from database.